dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPIConfigTest.java

```diff
@@ -89,7 +89,7 @@ public void setup() throws Exception {
     @After
     public void teardown() throws Exception {
         try {
-            server.stopServer();
+            server.stopServer("CWWKE0701E");
```

Comment:
This is pretty general.  I was thinking you would use regex on the specific message:

```java
"CWWKE0701E: .* BundleContext is no longer valid io.openliberty.microprofile.openapi.2.0.internal.servlet"
```